packages/react-components/react-tree/src/utils/tokens.ts

```diff
@@ -9,16 +9,16 @@ export const treeAvatarSize = {
 } as const;
 
 export const treeDataTypes: {
-  readonly [K in Uncapitalize<TreeOpenChangeData['type'] | TreeNavigationData_unstable['type']>]: Capitalize<K>;
+  readonly [K in TreeOpenChangeData['type'] | TreeNavigationData_unstable['type']]: K;
 } = {
-  arrowLeft: ArrowLeft,
-  arrowRight: ArrowRight,
-  enter: 'Enter',
-  click: 'Click',
-  expandIconClick: 'ExpandIconClick',
-  end: End,
-  home: Home,
-  arrowUp: ArrowUp,
-  arrowDown: ArrowDown,
-  typeAhead: 'TypeAhead',
+  ArrowLeft,
+  ArrowRight,
+  Enter: 'Enter',
```

Comment:
nit: You could import this

```suggestion
  Enter,
```